### ISSUE TITLE
Moving movie to movie name's first letter directory

### ISSUE THREAD

**Issue by queeup** [jkwill87/mnamer#44]
I was categorizing my movies with title's first letter on the directory.
ex: `/movies/B/B.Movies.1.mkv`
I was doing that with `{title:01}` on the old version. When I try to do that with `{name:01}` on the new version I am getting error:
```
(mnamer)  ~ $ mnamer -b -v tmp/mnamer-downloads-test/
Starting mnamer
loaded config from '/home/user/.mnamer-v2.json'

system
 - date = 2020-02-01
 - platform = Linux-4.15.0-74-generic-x86_64-with-elementary-5.1-hera
 - arguments = ['-b', '-v', 'tmp/mnamer-downloads-test/']
 - python version = 3.6.9
 - mnamer version = 2.0.0

settings
 - targets = [PosixPath('tmp/mnamer-downloads-test')]
 - batch = True
 - lower = False
 - recurse = False
 - scene = False
 - verbose = True
 - hits = False
 - ignore = ['.*sample.*']
 - mask = ['.avi', '.mp4', '.mkv']
 - no_cache = False
 - no_guess = False
 - no_style = False
 - movie_api = tmdb
 - movie_directory = /home/user/mnamer-test-directory/moved-files/{name:01}
 - movie_format = {original}
 - episode_api = tvdb
 - episode_directory = /home/user/mnamer-test-directory/moved-files/{series}
 - episode_format = {original}
 - version = False
 - config_dump = False
 - config_ignore = False
 - id_imdb = None
 - id_tmdb = None
 - id_tvdb = None
 - id_tvmaze = None
 - media = None
 - test = False
 - api_key_omdb = 477a7ebc
 - api_key_tmdb = db972a607f2760bb19ff8bb34074b4c7
 - api_key_tvdb = E69C7A2CEF2F3152
 - api_key_tvmaze = None
 - replacements = {'&': 'and', ':': '', ';': ',', '@': 'at'}

targets
 - /home/user/tmp/mnamer-downloads-test/Back.To.The.Future.1985.1080p.mp4

Processing Movie "Back.To.The.Future.1985.1080p.mp4"
/home/user/tmp/mnamer-downloads-test/Back.To.The.Future.1985.1080p.mp4
using tmdb

search parameters
 - extension = .mp4
 - group = None
 - quality = 1080p blu-ray h.264 aac
 - synopsis = None
 - media = movie
 - name = Back to the Future
 - year = 1985
 - id_imdb = None
 - id_tmdb = None


============================== CRASH REPORT BEGIN ==============================

--------------------------------- environment ----------------------------------

 - date = 2020-02-01
 - platform = Linux-4.15.0-74-generic-x86_64-with-elementary-5.1-hera
 - arguments = ['-b', '-v', 'tmp/mnamer-downloads-test/']
 - python version = 3.6.9
 - mnamer version = 2.0.0

--------------------------------- stack trace ----------------------------------

Traceback (most recent call last):
  File "/home/user/.virtualenvs/mnamer/lib/python3.6/site-packages/mnamer/__main__.py", line 32, in main
    run()
  File "/home/user/.virtualenvs/mnamer/lib/python3.6/site-packages/mnamer/__main__.py", line 136, in run
    if target.destination == target.source:
  File "/home/user/.virtualenvs/mnamer/lib/python3.6/site-packages/mnamer/target.py", line 101, in destination
    dir_head = format(self.metadata, str(self.directory))
  File "/home/user/.virtualenvs/mnamer/lib/python3.6/site-packages/mnamer/metadata.py", line 147, in __format__
    re_pattern, self._format_repl, format_spec or "{name} ({year})"
  File "/home/user/.virtualenvs/mnamer/lib/python3.6/re.py", line 191, in sub
    return _compile(pattern, flags).sub(repl, string, count)
  File "/home/user/.virtualenvs/mnamer/lib/python3.6/site-packages/mnamer/metadata.py", line 110, in _format_repl
    value = _MetaFormatter().vformat(format_string, None, self.as_dict)
  File "/usr/lib/python3.6/string.py", line 194, in vformat
    result, _ = self._vformat(format_string, args, kwargs, used_args, 2)
  File "/usr/lib/python3.6/string.py", line 247, in _vformat
    result.append(self.format_field(obj, format_spec))
  File "/home/user/.virtualenvs/mnamer/lib/python3.6/site-packages/mnamer/metadata.py", line 24, in format_field
    return format(v, f) if v else ""
ValueError: '=' alignment not allowed in string format specifier

=============================== CRASH REPORT END ===============================

Dang, it looks like mnamer crashed! Please consider filling an issue at
https://github.com/jkwill87/mnamer/issues along with this report.

(mnamer)  ~ $ 

```

**Comment by jkwill87** [comment]
That's an interesting use case and one that should probably be supported. I'll take a look, thanks!

**Comment by jkwill87** [comment]
This should be fixed in v2.0.2. Run `pip3 install -U mnamer` to update. The syntax is a little different to adhere to Python's format specifier syntax, [here's an example](https://jkwill87.gitbook.io/mnamer/formatting#nesting-directories) of how it works.

Thanks again.
